packages/theming/src/utils/getColor.ts

```diff
@@ -214,6 +214,7 @@ export const getColor = memoize(
       shade,
       colors: theme.colors,
       palette: theme.palette,
+      opacity: theme.opacity,
```

Comment:
realized during testing that changes to the theme's `opacity` were not breaking memoization as expected.